pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -69,10 +77,64 @@ func NewDeployer(cfg types.Config, labels map[string]string) *Deployer {
 	}
 }
 
+var sanityCheck = versionCheck
+
+// sanityCheck guarantees the kpt and kustomize versions are compatible with skaffold.
+func versionCheck(dir string) error {
+	kptCmd := exec.Command("kpt", "version")
+	out, err := util.RunCmdOut(kptCmd)
+	if err != nil {
+		return fmt.Errorf("running `kpt version` :%w", err)
```

Comment:
Can we make this error string more actionable. e.g.
fmt.Errorf("failed to determine kpt version due to %s. Try installing kpt again and try.", err)

Reply:
Done